frontend/src/concepts/pipelines/elyra/utils.ts

```diff
@@ -57,6 +57,7 @@ export const generateElyraSecret = (
         engine: 'Tekton',
         auth_type: 'KUBERNETES_SERVICE_ACCOUNT_TOKEN',
         api_endpoint: route,
+        public_api_endpoint: `${route}/pipelineRuns/${namespace}`,
```

Comment:
We need the dashboard route here 

for example:
api_endpoint: https:://<ds-pipeline-api-server>
public_api_endpoint: https://<rhods-dashboard>/pipelineRuns/<namespace>

Reply:
so would the public api endpoint just be /`pipelineRuns/<namespace>` Im confused on what rhods-dashboard would be?

Reply:
ah right -- you wanted the host name not the route name

@Gkrumbach07 can we use the origin of Dashboard + `/pipelineRuns/the-namespace`?

Reply:
correct for public_api_endpoint, we want hostname of RHODS Dashboard in place of route.
route would point to ds-pipeline API server